samples/apps/copilot-chat-app/webapp/src/components/chat/plan-viewer/PlanStepCard.tsx

```diff
@@ -42,18 +64,31 @@ const useClasses = makeStyles({
         width: '650px',
         fontSize: '12px',
     },
+    dialog: {
+        width: '398px',
+        '& button': {
+            marginTop: tokens.spacingVerticalL,
+            width: 'max-content',
+        },
+    },
 });
 
 interface PlanStepCardProps {
-    index: number;
-    step: IPlanStep;
+    step: any;
```

Comment:
Decided not to define the plan object in the front end since we're not extracting any additional fields for the view model. If we defined the Plan object on the front end, we'd have to keep it consistent with the backend since we have to send a valid plan object back, which adds overhead if the kernel model changes at all.

Reply:
can we note this in a comment then? `any` always scares me but if we think it's necessary here then it will be good for future folks to understand why.

Reply:
Done